test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -850,7 +850,7 @@
     namespace: measure-ns-0
   workloads:
   - name: 10Nodes
-    labels: [integration-test, fast]
+    labels: [fast]
```

Comment:
I think we should keep "RequiredAffinity" and "PreferredAffinity", as they run very different code paths.

But "RequiredAffinity" and "RequiredAntiAffinity" run similar code paths.

Reply:
We have 8 test cases related to interpodaffinity plugin: `SchedulingPodAntiAffinity`, `SchedulingPodAffinity`, `SchedulingPreferredPodAffinity`, `SchedulingPreferredPodAntiAffinity`, `SchedulingRequiredPodAntiAffinityWithNSSelector`, `SchedulingPreferredAntiAffinityWithNSSelector`, `SchedulingRequiredPodAffinityWithNSSelector`, `SchedulingPreferredAffinityWithNSSelector`. I thought we don't need all of test cases with namespace selector, isn't it true? Or we could just leave the four cases with pod affinity and disable the anti affinity tests?

I think we can also disable `MixedSchedulingBasePod` that tests the first four affinity options at once.

Reply:
I see. In that case, let's skip all the `WithNSSelector` versions.

And ok with removing `MixedScheduling`

Reply:
Done

Reply:
If there are no "obvious" test cases that can be cut, then we could also ask to extend the 10 minute limit for the periodic job.

Reply:
The test cases cut in this PR looks like the only "obvious" ones. If it still wouldn't reduce the testing time sufficiently, we should ask for limit extension. Especially, number of the test cases will grow over time so we will meet the timeout again.

Reply:
Reraise the same question, do we need to run scheduler-perf in two different master CI?

https://github.com/kubernetes/kubernetes/pull/127238#issuecomment-2339417320

Previous answer was-
> The thinking was that in CI there is time to spare and we might as well just run more tests.
https://github.com/kubernetes/kubernetes/pull/127238#issuecomment-2340753826

Got the idea, but I'm not sure if scheduler-perf should eat up all the remaining time until the timeout. 

Will we reduce the num of tests every time we hit the CI timeout again? (not only scheduler-perf will be growing) 
We'll increase the tests a lot for sure, but then how do we decide whether we should add integration label on new ones, and whether they're worth being added even by extending the whole timeout?
Ultimately, why not run scheduler-perf only in our CI, which could run more and bigger scenarios with more powerful machine, and never bother other tests?

Reply:
My understanding was that the 10min timeout was per package.

Reply:
Oh, then all good

Reply:
> My understanding was that the 10min timeout was per package.

Exactly, we're not affected by other packages timeouts. We just have our own limited to 10 minutes.

Reply:
Which is a bit "unfair": other SIGs have their integration tests spread out over many different packages and thus get to run longer overall without hitting that limit.